samples/Sentry.Samples.Maui/MainPage.xaml.cs

```diff
@@ -100,4 +100,8 @@ private async void OnFeedbackClicked(object sender, EventArgs e)
     {
         await Navigation.PushModalAsync(new SubmitFeedback());
     }
+
+    private void TapGestureRecognizer_OnTapped(object sender, TappedEventArgs e)
+    {
```

Comment:
This method doesn't do anything. Is it required?

Reply:
Yes - the hook is required